### PR TITLE
Docs: Readme

### PR DESCRIPTION
**Pull request by sashaphmn**
The previous link did not work

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ Copy the `.env.example` file to a new file named `.env`:
 cp .env.example .env
 ```
 
-You just need to activate your API key via the [Bandada Dashboard](https://bandada.pse.dev/groups) homepage. Then you need to copy & paste the key into the `ADMIN_API_KEY` environment variable.
+You just need to activate your API key via the [Bandada Dashboard](https://bandada.pse.dev/) homepage. Then you need to copy & paste the key into the `ADMIN_API_KEY` environment variable.
```
Comment (vplasencia) [inline README.md]:
```suggestion
You just need to activate your API key via the [Bandada Dashboard](https://app.bandada.pse.dev/) homepage. Then you need to copy & paste the key into the `ADMIN_API_KEY` environment variable.
```

**Review by vplasencia — CHANGES_REQUESTED** [review]
Hey @sashaphmn! Thank you very much. I left one comment with the correct link to get the API key.

**Comment by sashaphmn** [comment]
Hey. Always happy to help

**Comment by vplasencia** [comment]
Hey @sashaphmn could you update the PR with this link `https://app.bandada.pse.dev/` instead of `https://bandada.pse.dev/`?

**Comment by sashaphmn** [comment]
Done

**Review by vplasencia — APPROVED** [review]
Hey @sashaphmn! Looks great! Thank you very much 🙏

**Pull request merged**